jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/EcsExecUtils.kt

```diff
@@ -87,4 +97,53 @@ object EcsExecUtils : CoroutineScope by ApplicationThreadPoolScope("EcsExec") {
             }
         )
     }
+
+    fun checkRequiredPermissions(project: Project, clusterArn: String, taskArn: String) {
```

Comment:
There is a lot of branching in here and 0 tests, please add some

Reply:
Tests added